e2e/data/smoke/block_mount.yaml

```diff
@@ -0,0 +1,20 @@
+apiVersion: v1
+kind: Pod
+metadata:
+  name: block-test
+spec:
+      containers:
+      - image: busybox
+        name: block-test1
```

Comment:
our automated test is looking to write data into /tmp/rook1. this was not a test change.